metadata/water/lakes_nhd

```diff
@@ -0,0 +1,85 @@
+
+This dataset represents bodies of water as polygons that include attributes such as the area in square kilometers and [GNIS](https://www.usgs.gov/tools/geographic-names-information-system-gnis) name. UGRC has also modified the source NHD data to create additional functionality. Four fields have been added to this feature class (FType_Text, FCode_Text, InUtah and IsMajor) and features have been split at the state boundary.
+
+The "IsMajor" field identifies major lakes for coarse scale mapping and display, where 1 = major and 0 = minor. The "InUtah" field identifies whether a given body of water is within the state boundary, where 1 = in Utah and 0 = outside of Utah. For a full description of the fields in this dataset, please see the [schema summary](https://docs.google.com/document/d/1QWZKKPe3PWR9XyMYEG6qFmhsK55oBYYLWRQtknbrnD4/edit?usp=sharing).
+
+### How was the dataset created?
+
+The NHD was first created in the late 1990s by USGS in collaboration with the Environmental Protection Agency [(EPA)](https://www.epa.gov/). The original product, the National Hydrography Dataset, was created at a 1:100,000 scale for the conterminous United States. The EPA became the primary custodian of this medium resolution dataset in the early 2000s, while USGS and United States Forest Service [(USFS)](https://www.fs.usda.gov/) began the production of what would become the high resolution National Hydrography Database. Today, the high-resolution NHD is available at a nationwide 1:24,000 scale.
+
+More information on the history of this database can be found in [publications](https://www.horizon-systems.com/NHDPlusData/NHDPlusV21/Documentation/History/Making_the_Digital_Water_Flow.pdf) from USGS and on the [NHD website](https://www.usgs.gov/national-hydrography/national-hydrography-dataset#:~:text=In%20the%20late%201990s%2C%20the,and%20those%20of%20other%20medium).
+
+### How reliable and accurate is the dataset?
+
+The NHD is the primary source for water body data in the United States. It represents a nationally consistent, interoperable dataset that facilitates accurate data sharing between entities. Questions or concerns about this dataset can be directed to [USGS](https://www.usgs.gov/national-hydrography/nhdplus-high-resolution). Utah-specific inquiries can be directed to [our team](https://gis.utah.gov/contact/) at UGRC.
+
+## Credits
+
+- USGS
+- EPA
+- UGRC
+
+### Data Source
+
+NHD
+
+### Host
+
+UGRC
+
+## Restrictions
+
+## License
+
+## Tags
+
+- Water
+- National Hydrography Dataset
+- USGS
+- EPA
+- Hydrology
+
+## Secondary Category
+
+## Data Page Link
+
+[https://gis.utah.gov/products/sgid/water/nhd-lakes/]
+
+## Update
+
+### Update Schedule
+
+<!--- The SGID Index does not specify how often this layer is updated. Moreover, the USGS website says that the NHD was retired in 2023 and will no longer be maintained. It says that the 3D Hydrography Program will contain the most recent data. Is this something we want to include in our metadata? 
```

Comment:
**question**: as far as i know zach updated it as needed. I'm not sure I was aware that it was retired in 2023 and we should absolutely be migrating to the newest data. @ZachBeck and @gregbunce is this something you are aware of and planning for?